src/test/java/uk/ac/ebi/ampt2d/accession/generator/TestsGeneratorConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.accession.generator;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.ac.ebi.ampt2d.accession.AccessionGenerator;
+
+@Configuration
+public class TestsGeneratorConfiguration {
```

Comment:
In other EVA projects this kind of configuration classes are placed in [a `test.configuration` package](https://github.com/EBIvariation/eva-pipeline/tree/develop/src/test/java/uk/ac/ebi/eva/test/configuration) to avoid mixing them with the main tests.